sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -80,7 +80,7 @@ private static IReadOnlyList<CustomFormSubModel> ConvertFromUnlabeled(KeysResult
                 {
                     string fieldName = "field-" + i;
                     string fieldLabel = cluster.Value[i];
-                    fieldMap.Add(fieldName, new CustomFormModelField(fieldName, fieldLabel, default));
+                    fieldMap.Add(fieldName, new CustomFormModelField(fieldName, fieldLabel, 0.0f));
```

Comment:
temporary. This is fixed in PR https://github.com/Azure/azure-sdk-for-net/pull/11363